Makefile-docker

```diff
@@ -36,6 +36,7 @@ jquery.cookie/jquery.cookie.js \
 jszip/dist/jszip.js \
 timeago/jquery.timeago.js \
 underscore/underscore.js \
+netmask/lib/netmask.js \
```

Comment:
do you need to update `settings.MINIFY_BUNDLES` too?

Reply:
No, we don't have a bundle for admin js/css - they get included through the `Media` class on the `ModelAdmin`, which is good enough - we don't get minification but we do get a unique filename thanks to `staticfiles`.